docs/src/main/asciidoc/cache.adoc

```diff
@@ -597,6 +597,36 @@ public class CacheKeysService {
 }
 ----
 
+=== Populating a `CaffeineCache`
+
+You can populate a `CaffeineCache` using the `CaffeineCache#put(Object, CompletableFuture)` method.
+This method associates the `CompletableFuture` with the given key in the cache. If the cache previously contained a value associated with the key, the old value is replaced by this `CompletableFuture`. If the asynchronous computation fails, the entry will be automatically removed.
+
+[source,java]
+----
+package org.acme.cache;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import io.quarkus.cache.Cache;
+import io.quarkus.cache.CacheName;
+import io.quarkus.cache.CaffeineCache;
+
+import java.util.concurrent.CompletableFuture;
+
+@ApplicationScoped
+public class CacheKeysService {
```

Comment:
The preceding code snippet used that class name because it was about retrieving the keys of a cache. Let's get rid of that part of the class name.

```suggestion
public class CacheService {
```